### PR TITLE
Bug/delete app cleanup

### PR DESCRIPTION
**Pull request by connectbo**
This pr is intended to clean up messages in the side drawer and stop polling when an instance is stopped.

related: https://github.com/helxplatform/development/issues/839

### CODE REVIEW

**Comment by connectbo** [comment]
@mbwatson I think this pr is ready for review. When an instance is not up and ready but you want to delete it, its polling function will break and the 'xxx is launching' message will be removed from the side drawer. I got this deployed in my [namespace](https://helx-dev-zbo.apps.renci.org/helx) if you want to click around. Feel free to share your thoughts.

File: src/contexts/instance-context.js [hunk 1]
```diff
@@ -10,13 +10,21 @@ export const InstanceProvider = ({ children }) => {
     const { helxAppstoreUrl } = useEnvironment();
     const { addActivity, updateActivity } = useActivity();
     const [openedTabs, setTabs] = useState([]);
+    const [pollingTimerIDs, setpollingTimerIDs] = useState([])
```
Comment (mbwatson) [inline src/contexts/instance-context.js:13]: teeny thing, but i think you wanted a capital "P" in setpollingTimerIDs

**Review by mbwatson — APPROVED** [review]
it works! and this is a nice solution.

File: src/contexts/instance-context.js (hunk 1 above)
Comment (connectbo) [reply src/contexts/instance-context.js:13]: Nice catch on this! Right, definitely should be a capital 'P' there, will change that. Thanks.

**Pull request merged**